server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -823,6 +824,29 @@ public void getMessageListShouldSortMessagesWhenSortedByDateAsc() throws Excepti
             .body(ARGUMENTS + ".messageIds", contains(message1.getMessageId().serialize(), message2.getMessageId().serialize()));
     }
 
+    @Test
+    @Ignore
+    public void getMessageListShouldSortUsingInternalDateWhenNoDateHeader() throws Exception {
```

Comment:
Already the case. ES indexes internalDate as SentDate when none can be located:

```
 String sentDate = DateResolutionFormater.DATE_TIME_FOMATTER.format(headerCollection.getSentDate().orElse(internalDate));
```

(from IndexableMessage line 141)